src/rollup/RollupCreator.sol

```diff
@@ -113,6 +120,20 @@ contract RollupCreator is Ownable {
                 validatorWalletCreator: validatorWalletCreator
             })
         );
+        {
+            IRollupAdmin rollupAdmin = IRollupAdmin(address(rollup));
```

Comment:
if you remove this var and just use these below
```
IRollupAdmin(address(rollup)).setValidator(...)
and
IRollupAdmin(address(rollup)).setOwner(...)
```
it should fit into the stack without scoping

edit: you also need to fix the `proxyAdmin.transferOwnership(tempOwner);`